tms/views/assets.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
 
-<template id="assets_backend" name="tms assets" inherit_id="web.assets_backend">
-    <xpath expr="." position="inside">
-        <link rel="stylesheet" href="/tms/static/src/css/kanban_view.scss" />
-    </xpath>
-</template>
+    <template id="assets_backend" name="tms assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <link rel="stylesheet" href="/tms/static/src/css/kanban_view.scss" />
+        </xpath>
+    </template>
 
 </odoo>
```

Comment:
This file should be declared at manifest 

See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0 8th point